docs/src/generic_hybrid.md

```diff
@@ -67,12 +57,9 @@ hybrid_model = constructHybridModel(
     scale_nn_outputs = true, # Scale neural network outputs
     input_batchnorm = true   # Apply batch normalization to inputs
 )
-```
-
-### 5. Train the Model
 
-```@example quick_start_training
-using WGLMakie # to see an interactive and automatically updated train_board figure
+# 5. Train the Model
+using CairoMakie # to see an interactive and automatically updated train_board figure
```

Comment:
we still not support this, saving the animation. So, no animation for now, we could comment out this line, for now. I also need to update the progress bar issue.